lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -223,7 +223,7 @@ def add_async_options(parser):
 
 def add_basedir_options(parser):
     """Add options for commands which can set a playbook basedir"""
-    parser.add_argument('--playbook-dir', default=None, dest='basedir', action='store',
+    parser.add_argument('--playbook-dir', default=os.environ.get('ANSIBLE_PLAYBOOK_DIR', None), dest='basedir', action='store',
```

Comment:
make it a full entry (type: path) in config/base.yml as the next thing 'missing' will be setting it in ansibe.cfg

Reply:
fair enough